### PR TITLE
polite way to greet someone

### PR DESCRIPTION
**Pull request by Kevsani**

### CODE REVIEW

**Comment by jdfreder** [comment]
Hi kevin!

File: ping.html [hunk 1]
```diff
@@ -11,6 +11,7 @@
     </script>
 </head>
 <body>
+	<h2>HI</h2>
```
Comment (jdfreder) [inline ping.html:14]: This is pointless, but good GIT practice!

Comment (jdfreder) [inline ping.html:14]: :hankey:

Comment (Kevsani) [inline ping.html:14]: I agree! :hankey:

**Pull request closed without merging**